### PR TITLE
update milvus readme with milvus lite

### PR DESCRIPTION
**Pull request by zc277584121**

### CODE REVIEW

**Comment by zc277584121** [comment]
Hello @bilgeyucel , here is a synchronize with our [new milvus-haystack change](https://github.com/milvus-io/milvus-haystack/pull/19), which introduct the new feature of milvus-lite which is about to be released.

File: integrations/milvus-document-store.md [hunk 1]
```diff
@@ -33,60 +34,97 @@ toc: true
...
-pip install -U milvus-haystack
+
+```shell
+pip install --upgrade pymilvus milvus-haystack
 ```
 
 ### Usage
 
-First, to start up a Milvus service, follow the ['Start Milvus'](https://milvus.io/docs/install_standalone-docker.md#Start-Milvus) instructions in the documentation. 
+By default, if you install the latest version of pymilvus, you don't need to start the milvus service manually.
```
Comment (bilgeyucel) [inline integrations/milvus-document-store.md:44]: Can you give information about the latest version number here? Is it 2.4+?

**Review by bilgeyucel — COMMENTED** [review]
Thanks @zc277584121, I left one minor comment

File: integrations/milvus-document-store.md (hunk 1 above)
Comment (zc277584121) [reply integrations/milvus-document-store.md:44]: Yes, it is ,  2.4+ is ok,  2.4.3+ is more stable

**Comment by zc277584121** [comment]
I have updated the description, which I think is a better expression. @bilgeyucel could you please help to review it

File: integrations/milvus-document-store.md [hunk 2]
```diff
@@ -33,60 +34,92 @@ toc: true
...
 
 ### Usage
 
-First, to start up a Milvus service, follow the ['Start Milvus'](https://milvus.io/docs/install_standalone-docker.md#Start-Milvus) instructions in the documentation. 
+Use the `MilvusDocumentStore` in a Haystack pipeline as a quick start.
+
+```python
+from haystack import Document
+from milvus_haystack import MilvusDocumentStore
+
```
Comment (annthurium) [inline integrations/milvus-document-store.md:49]: Running into this code, there's a dependency error:

```
ContextualVersionConflict                 Traceback (most recent call last)
[<ipython-input-2-678eb02a0f80>](https://localhost:8080/#) in <cell line: 2>()
      1 from haystack import Document
----> 2 from milvus_haystack import MilvusDocumentStore
      3 
      4 document_store = MilvusDocumentStore(
      5     connection_args={"uri": "./milvus.db"},  # Milvus Lite

8 frames
[/usr/local/lib/python3.10/dist-packages/pkg_resources/__init__.py](https://localhost:8080/#) in _resolve_dist(self, req, best, replace_conflicting, env, installer, required_by, to_activate)
    871             # Oops, the "best" so far conflicts with a dependency
    872             dependent_req = required_by[req]
--> 873             raise VersionConflict(dist, req).with_context(dependent_req)
    874         return dist
    875 

ContextualVersionConflict: (grpcio 1.64.0 (/usr/local/lib/python3.10/dist-packages), Requirement.parse('grpcio<=1.63.0,>=1.49.1'), {'pymilvus'})
```

@zc277584121 any ideas on how to resolve? This was in a Colab, in a clean Python environment where I ran only the pip commands suggested in this documentation.

Comment (zc277584121) [reply integrations/milvus-document-store.md:49]: In Colab, after running the pip command, we have to restart the session kernel to make the grpcio dependency really installed. I think it is a problem in Colab, where the default environment is not clean but exist an installed grpcio. And after pip install pymilvus[->grpcio], actually it is not be install and can not work normally. and we have to restart the colab kernel to make the `pip install xxx` really be installed and work normally.

Comment (zc277584121) [reply integrations/milvus-document-store.md:49]: here is our official document: https://milvus.io/docs/integrate_with_haystack.md , where we mind users to restart kernel in colab
image

Comment (anakin87) [reply integrations/milvus-document-store.md:51]: I added a suggestion in this page, too

**Review by anakin87 — APPROVED** [review]
The examples work and look good.

Thank you!

**Pull request merged**